src/neo/Wallets/NEP6/NEP6Wallet.cs

```diff
@@ -300,5 +301,32 @@ public override bool VerifyPassword(string password)
                 }
             }
         }
+
+        public bool ChangePassword(string password_old, string password_new)
+        {
+            bool isSuccessful = true;
+            lock (accounts)
+            {
+                Parallel.ForEach<NEP6Account>(accounts.Values, (account, state) =>
+                {
+                    if (!account.ChangePassword(password_old, password_new))
+                    {
+                        state.Stop();
+                        isSuccessful = false;
```

Comment:
If the password of one of the accounts is not changed successfully, the passwords in the wallet will be inconsistent. Then the password may not be able to be changed properly permanently.

Reply:
Have rewritten password changing strategy, this shhould have been resolved:)